src/Explorer/Notebook/NotebookUtil.ts

```diff
@@ -328,4 +329,21 @@ export class NotebookUtil {
     link.click();
     document.body.removeChild(link);
   }
+  public static getDownloadModalConent(fileName: string): string {
+    if (this.isPhoenixEnabled()) {
+      return `In order to download and run your notebook, a local temporary environment will be created for your account to view, edit and run the notebooks for a dedicated period of time. If you need the notebook to be stored permanently, please connect to github and then download the notebook to your github repo.`;
```

Comment:
To download, run, and make changes to this sample notebook, a temporary workspace will be created. 

When the session expires, any notebooks in the workspace will be removed. To save your work permanently, save your notebooks to a GitHub repository or download the notebooks to your local machine before the session ends. Learn more.